Andrew

amroczk4 opened the pull request.

amroczk4 committed:
Added non-admin functionality

new users can register new account
non-admin users can now log in

amroczk4 committed:
Update README.md

amroczk4 committed:
Update README.md

mattcongreve-depaul committed:
updated gitignore

mattcongreve-depaul committed:
added gitignore edit

mattcongreve-depaul committed:
added gitignore edit 2

mattcongreve-depaul committed:
deleted idea folder

mattcongreve-depaul committed:
removed pyc files

mattcongreve-depaul commented on README.md:
```diff
@@ -2,9 +2,13 @@
 https://youtu.be/eosXCVcgtPw
+
+Required Installations:
+- pip install crispy-bootstrap5
```
I recommend putting these in a `requirements.txt`, which you can generate with `pip freeze > requirements.txt` . Then document that the way we want to install the packages is with `pip install -r requirements.txt`

amroczk4 committed:
Updated README.md

Updated README and added requirements.txt

amroczk4 committed:
Minor changes for readability to views.py

amroczk4 committed:
Added Admin Panel Link

When admin users log into the homepage, a navbar link will appear to direct them to the admin panel

mattcongreve-depaul commented:
approved
